modules/apiserversource-yaml.adoc

```diff
@@ -236,10 +273,34 @@ Data,
 [id="delete-apiserversource-yaml_{context}"]
 == Deleting the ApiServerSource
 
-You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `oc` commands:
+. Delete the trigger by entering the following command:
```

Comment:
You can remove "by entering the following command" from each of these steps. For example, "Delete the trigger:"